### PR TITLE
Mining Medics

### PR DESCRIPTION
**Pull request by Super3222**
#### Intent of your Pull Request

Re-introducing Mining Medics: A new position in cargo with the sole objective to keep up to date on the miners well being- the mining medic spends their time at a short ended office within the premises of the mining station.

This is a single person role with a simple basis geared on the basis of putting a paramedic on lavaland. This also constitutes as a rework of the older form of a Mining Medic, which was practically a buddy that followed other miners around the asteroid.

Mining Medics are equipped with special uniforms that contain bio resistance (credit to KMC2000 for the sprite - made about 3-4 or so months ago).
Their locker holds a hand held crew monitor, medical supplies, an alternative costume, and more.

To the trained eye, their office is lacking a few things here and there that could benefit the miners a lot. Via a cloning machine, cyro tube, or even a surgery table. Albeit, those things do sound nice - that's not what the Mining Medic is for. I am not adding an entire medbay section to the mining station.

While the Mining Medic surely restores people back to health with their sleeper, something like cloning or implanting legion souls has to be handled back on the station. So not every last thing gets knotted down, and lavaland is still treacherous. I am interested in adding at least a defibrillator somewhere in their office however.

Disclaimer: This was a pretty old project that I moved on from in the past, but I've found the time to make the final adjustments here and there to make it at least presentable and hopefully it'll be brought into the game for players to enjoy.

:cl: Super3222
rscadd: INTRODUCING A NEW JOB - MINING MEDICS
rscadd: Paramedics of the lavaland, and doctors on the mining station. Their sworn duty is to help injured or decease miners back to health.
rscadd: They have an office on the left wing of the mining station with supplies, and a crew monitor computer which they can use to keep an eye on the miners at all times.
rscadd: They also get ATV's!!!
rscadd: Credit to KMC2000 for spriting their outfit.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
where's the mining medic hood you grr

**Comment by matskuman5** [comment]
Give them defibs

**Comment by Time-Green** [comment]
Give them a mop. In-case a miner is a little to broken.

**Comment by Super3222** [comment]
This is ready for review.

**Comment by ShadowDeath6** [comment]
where is the hood at

**Comment by Super3222** [comment]
In the last commit(s):
- Mining Medics now have access to the morgue.
- Mining Medics do NOT have access to the station's surgery room.
- Mining Medics STILL have their mining access. That commit message was incorrect

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -1005,8 +1005,8 @@
 		reagents.add_reagent("cream", rand(2, 5))
 	if(prob(45))
 		reagents.add_reagent("salglu_solution", rand(2,5))
-	if(prob(8))
-		reagents.add_reagent("omnizine", rand(10,15))
+	if(prob(5))
```
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: secret nerf grr

File: code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -130,6 +130,13 @@
 	item_state = "headset"
 	keyslot = new /obj/item/device/encryptionkey/headset_medsci
 
+/obj/item/device/radio/headset_medcargo
```
Comment (Cruix) [inline code/game/objects/items/devices/radio/headset.dm]: This needs to be /obj/item/device/radio/headset/medcargo

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 3]
```diff
@@ -106,18 +106,21 @@
 
 /obj/structure/closet/secure_closet/mmedical/New()
 	..()
-	sleep(2)
 	if(prob(50))
 		new /obj/item/weapon/storage/backpack/medic(src)
 	else
 		new /obj/item/weapon/storage/backpack/satchel_med(src)
+	new /obj/item/weapon/defibrillator(src)
```
Comment (Cruix) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: This needs to be /obj/item/weapon/defibrillator/loaded

File: code/modules/jobs/job_types/medical.dm [hunk 4]
```diff
@@ -184,37 +184,37 @@ Virologist
...
 	selection_color = "#ffeef0"
 
 	outfit = /datum/outfit/job/miningmedic
 
-	access = list(access_medical, access_morgue, access_surgery, access_maint_tunnels, access_mailsorting, access_cargo, access_cargo_bot, access_qm, access_mint, access_mining, access_mining_station, access_mineral_storeroom)
-	minimal_access = list(access_medical, access_mining, access_mint, access_mining_station, access_mailsorting, access_mineral_storeroom)
-
-*/
+	access = list(access_medical, access_morgue, access_surgery, access_cargo, access_mint, access_mining, access_mining_station, access_mineral_storeroom)
+	minimal_access = list(access_medical, access_morgue, access_mining, access_mint, access_mining_station, access_mailsorting, access_mineral_storeroom)
```
Comment (Cruix) [inline code/modules/jobs/job_types/medical.dm:202]: access_cargo is for the cargo bay, access_mailsorting is for the cargo front office. Both should have access_mailsorting, and minimal access at least should not have access_cargo.

**Review by Cruix — CHANGES_REQUESTED** [review]

**Comment by Cruix** [comment]
Also you did not push the .dmi changes, so there are no new icons.

**Comment by Super3222** [comment]
A vigilant mining medic enjoys his first and last glimpses of calmness before the miners find their way to anarchy.

**Comment by ShadowDeath6** [comment]
Shower > washing machine

**Comment by Super3222** [comment]
The room's not getting a shower though. Washing machine gives it an at-home feel which is what the room is trying to accomplish.

**Comment by ShadowDeath6** [comment]
You say as it's a dirty room with high level medical equipment and metal floors

**Comment by Super3222** [comment]
Yeah, and there's more people who play this game than just you, Shadow. I could care less about putting a shower there instead of a washing machine. I think a washing machine is nice, so I put it there. I'm not looking for what cleans your dirty shoes the fastest.

**Comment by ShadowDeath6** [comment]
washing machine needs a buff imho

**Review by Cruix — APPROVED** [review]

**Comment by Super3222** [comment]
@Cruix Can this be merged, and updated into the game today? I missed that opportunity by a second this morning after being pushed off a cliff by map merges, and this is my oldest PR still open.

**Comment by Cruix** [comment]
Yes just let me make sure the atvs work

**Pull request merged**